docs/operator-guide/cd/deploy-application-in-remote-cluster-via-irsa.md

```diff
@@ -0,0 +1,683 @@
+---
+
+title: "Deploy Application In Remote Cluster via IRSA"
+sidebar_label: "Deploy Application In Remote Cluster via IRSA"
+description: "Learn how to securely deploy applications across AWS accounts using IRSA (IAM Roles for Service Accounts) in KubeRocketCI. Step-by-step guide for EKS cross-account access configuration."
+
+---
+<!-- markdownlint-disable MD025 -->
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Deploy Application In Remote Cluster via IRSA
+
+<head>
+  <link rel="canonical" href="https://docs.kuberocketci.io/docs/operator-guide/cd/deploy-application-in-remote-cluster-via-irsa/" />
+</head>
+
+KubeRocketCI provides the capability to deploy applications securely using IAM Roles for Service Accounts (IRSA) in AWS EKS.
+This integration enables Kubernetes pods to assume IAM roles for secure and temporary access to AWS resources, eliminating the need for long-lived credentials. While the deployment process is streamlined for most users, the platform also supports advanced configurations for custom permissions and role management, ensuring flexibility for more complex scenarios.
+
+## Prerequisites
+
+To start using this approach, you need to have OIDC (OpenID Connect) already configured for your EKS cluster. This setup allows Kubernetes service accounts to securely assume IAM roles. For your convenience, please follow our documentation and use the preconfigured deployment code for clusters available in our repository:
```

Comment:
I think we have documentation on enabling IRSA, please add link here and not to the terraform code